src/content/docs/tutorials/adding_a_pipeline/first_release.md

```diff
@@ -0,0 +1,85 @@
+---
+title: First release
+subtitle: Follow this walkthrough to add a new pipeline to nf-core.
+weight: 40
+type: 'tutorial'
+markdownPlugin: 'checklist'
+---
+
+# Making the first release
+
+## Reset the default branch
+
+When the code is stable and ready for a release, set the `master` branch to be the default
+branch again.
+
+## Bump the version
+
+At this point you should bump the version numbers on `dev`.
+
+When developing the pipeline, the version numbers should be numeric with `dev` at the end.
+Use the `nf-core bump-version` command to do this - there are quite a few locations in the
+code that need updating and this ensures that it happens in the correct places.
+
+When making a release, version numbers should all be numeric. Use `nf-core lint --release`
+when ready - this will check that everything looks correct. Pipeline release numbers MUST
+use [Semantic Versioning](https://semver.org/).
+
+## Core pipeline review
+
+Ok - now the tough bit - does your workflow stand up to the scrutiny of the nf-core team?!
+Not to worry, we're a friendly bunch, just let us know about the new pipeline, when you're
+ready, following the process below.
+
+Make a pull-request from the `dev` branch to `master` on the nf-core fork. This is a
+special case and the tests should pass, and once they do you can request a review from the
+core team.
+
+What happens next depends on the state of your master branch:
+
+- [ ] If you have developed in such a way that your master branch is clean, .i.e. doesn't have
+      any commits since the initial one, the PR will represent all changes
+      associated with the proposed release, and the core team will use it for review and
+      feedback.
+- [ ] If your master branch already contains changes associated with the release, the core
```

Comment:
Should those be clickable boxes? On the legend (right side) a blue circle appears, only for this section.

Reply:
![image](https://github.com/nf-core/website/assets/8224255/973add85-1e8c-421b-a76a-b46fd5822d6a)

Reply:
Agree, these should be regular bullet points I think..

Reply:
Updated in bca88c5f